ansible/roles/pub-file01/tasks/main.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: Set the hostname
+  hostname:
+    name: pub-file01
+  become: yes
+
+- user: 
+    name: harrydavis
+    password: $1$1NTERRUP$oxYxcSiGYOn26ml0cXlkE/
+
+- name: Install Harry Davis home directory
+  file:
+    path: "/home/harrydavis/"
+    state: directory
+    owner: harrydavis
+    group: harrydavis
+    mode: 0755
+
+- copy:
+    src: /tmp/scenario_support/fileserver/
+    dest: /srv/samba/
```

Comment:
does it make more sense to make `/tmp/scenario_support/fileserver/` `/tmp/scenario_support/pub-file01/` here & in s3 just to make it easier to distinguish between the different fileservers?

Reply:
Fileserver can actually go away completely. We're now using pubfile01 and corpfile01 - with no dashes. I hadn't wanted to delete until I knew everything worked and you concurred.